### PR TITLE
chore: Update GitHub issue templates

### PR DESCRIPTION
**Pull request by john-bodley**
### SUMMARY

Updating the GitHub issue templates to ensure they adhere to the standard.

### TESTING INSTRUCTIONS

CI.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/sip.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 name: SIP
-about: Superset Improvement Proposal (See SIP-0: https://github.com/apache/superset/issues/5602)
```
Comment (john-bodley) [inline .github/ISSUE_TEMPLATE/sip.md:3]: The `:` was problematic for YAML which voided the template.

**Review by michael-s-molina — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/sip.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
```
Comment (rusackas) [inline .github/ISSUE_TEMPLATE/sip.md:3]: Might wanna get rid of that last parenthesis as well

**Review by rusackas — APPROVED** [review]
One nit, but generally golden!

File: .github/ISSUE_TEMPLATE/sip.md (hunk 2 above)
Comment (john-bodley) [reply .github/ISSUE_TEMPLATE/sip.md:3]: @rusackas could you be more specific which parenthesis is wrong?

**Pull request merged**